### PR TITLE
Added casting to a float for the scheduler service to prevent error if s...

### PR DESCRIPTION
**Pull request by seman**
...tr is passed to the create_interval_timer() method. Resolved issue 1008

### CODE REVIEW

File: ion/services/cei/test/test_scheduler.py [hunk 1]
```diff
@@ -422,6 +422,7 @@ def cb(*args, **kwargs):
         self.addCleanup(sub.stop)
 
         tid = self.ssclient.create_interval_timer(start_time="now",
+                                                  end_time=-1,
```
Comment (lukecampbell) [inline ion/services/cei/test/test_scheduler.py]: shouldn't this be `end_time='-1'`?

**Comment by lukecampbell** [comment]
If you could just change line 425 of ion/services/cei/test/test_scheduler.py we'll be all set to merge and close [OOIION-1008](https://jira.oceanobservatories.org/tasks/browse/OOIION-1008)

**Comment by seman** [comment]
I made the changes.

**Comment by mmeisinger** [comment]
Should this be merged?

**Comment by mmeisinger** [comment]
Is this ready to be merged?

**Comment by seman** [comment]
I am not sure why I am not getting a notice when you commented here. 

Yes, this was ready to be merged. Can you still merge  it?

**Comment by mmeisinger** [comment]
Thanks Seman. It is not automatically mergeable. Do you know why?

**Comment by seman** [comment]
Not sure the reason. It was auto merge when I pushed it. I will take look at it.

**Comment by seman** [comment]
I resolved the merge conflict in _bootstrap_qc_post_processor.py_. Try it now.

**Comment by mmeisinger** [comment]
Thanks, looks good :+1:

**Pull request merged**